### PR TITLE
bulk change to unify **Required.** and **Optional.**

### PR DESCRIPTION
**Pull request by pelszkow**
### Details:
 - *bulk change to unify \*\*Required\*\* and \*\*Optional\*\**
 - *\*\*Required\*\*. -> \*\*Required.\*\**
 - *\*\*Required\*\*: \*Yes\* -> \*\*Required\*\*: \*yes\**
 - side effects - remove some whitespaces. 

### Tickets:
 - *59113*

### CODE REVIEW

**Comment by ggalieroc** [comment]
LGTM. But since this [comment](https://github.com/openvinotoolkit/openvino/pull/6477/files#r662978725) I think we should double check with doc maintainers.

**Comment by ggalieroc** [comment]
@tsavina could you please help us to set the right way to indicate Required and Optional inputs/outputs in op spec?
We are trying to unify all specs as different ones use different approach.
Thread [here](https://github.com/openvinotoolkit/openvino/pull/6477/files#r662978725)

**Review by ggalieroc — APPROVED** [review]

**Comment by pelszkow** [comment]
Mentioned thread was close so I ask once again here.
How we should mark `required` (where put the dot):
1) `**Required.**`
2) `**Required**.`
in docs/ops/ more often we use 1st approach 180/38

@lazarevevgeny, @openvinotoolkit/openvino-docs-maintainers

**Comment by ilyachur** [comment]
> Mentioned thread was close so I ask once again here.
> How we should mark `required` (where put the dot):
> 
> 1. `**Required.**`
> 2. `**Required**.`
>    in docs/ops/ more often we use 1st approach 180/38
> 
> @lazarevevgeny, @openvinotoolkit/openvino-docs-maintainers

I prefer to have the first variant  `**Required.**` because it is one sentance. @rkazants What do you think?

**Comment by rkazants** [comment]
> > Mentioned thread was close so I ask once again here.
> > How we should mark `required` (where put the dot):
> > 
> > 1. `**Required.**`
> > 2. `**Required**.`
> >    in docs/ops/ more often we use 1st approach 180/38
> > 
> > @lazarevevgeny, @openvinotoolkit/openvino-docs-maintainers
> 
> I prefer to have the first variant `**Required.**` because it is one sentence. @rkazants What do you think?

I agree with @ilyachur, `**Required.**` looks better.

**Comment by pelszkow** [comment]
Now PR unify to `**Required.**`.

File: docs/ops/movement/SpaceToDepth_1.md [hunk 1]
```diff
@@ -49,15 +28,32 @@ If `mode = depth_first`:
...
+
+*SpaceToDepth* operation permutes element from the input tensor with shape `[N, C, D1, D2, ..., DK]`, to the output tensor where values from the input spatial dimensions `D1, D2, ..., DK` are moved to the new depth dimension. Refer to the [ONNX* specification](https://github.com/onnx/onnx/blob/master/docs/Operators.md#SpaceToDepth) for an example of the 4D input tensor case.
+
+The operation is equivalent to the following transformation of the input tensor `data` with `K` spatial dimensions of shape `[N, C, D1, D2, ..., DK]` to *Y* output tensor. If `mode = blocks_first`:
+
+    x' = reshape(data, [N, C, D1/block_size, block_size, D2/block_size, block_size, ... , DK/block_size, block_size])
+
+    x'' = transpose(x',  [0,  3, 5, ..., K + (K + 1), 1,  2, 4, ..., K + K])
 
-**Types**
```
Comment (mitruska) [inline docs/ops/movement/SpaceToDepth_1.md:58]: This file contains some probably merge artifacts like removing **Types**, while this section has been added during SpecToDepth spec update.

File: docs/ops/movement/SpaceToDepth_1.md [hunk 2]
```diff
@@ -49,15 +28,32 @@ If `mode = depth_first`:
 
 **Inputs**
 
-*   **1**: `data` - input tensor of type *T* with rank >= 3. **Required.**
```
Comment (mitruska) [inline docs/ops/movement/SpaceToDepth_1.md:52]: "type \*T\*" shouldn't be changed

File: docs/ops/movement/SpaceToDepth_1.md [hunk 3]
```diff
@@ -49,15 +28,32 @@ If `mode = depth_first`:
...
+
+If `mode = depth_first`:
+
+    x' = reshape(data, [N, C, D1/block_size, block_size, D2/block_size, block_size, ..., DK/block_size, block_size])
+
+    x'' = transpose(x', [0,  1, 3, 5, ..., K + (K + 1),  2, 4, ..., K + K])
+
+    y = reshape(x'', [N, C * (block_size ^ K), D1 / block_size, D2 / block_size, ..., DK / block_size])
 
-* *T*: any supported type.
```
Comment (mitruska) [inline docs/ops/movement/SpaceToDepth_1.md:60]: This line shouldn't be removed

**Review by mitruska — COMMENTED** [review]
Please update this PR changes to the revised version of SpaceToDepth spec.

**Review by mitruska — APPROVED** [review]

**Review by ilyachur — APPROVED** [review]

**Pull request merged**